fcrepo-http-api/src/main/java/org/fcrepo/http/api/ContentExposingResource.java

```diff
@@ -602,10 +585,16 @@ protected void replaceResourceWithStream(final FedoraResource resource,
     }
 
     protected void patchResourcewithSparql(final FedoraResource resource,
-                                           final String requestBody,
-                                           final RdfStream resourceTriples)
-            throws MalformedRdfException, AccessDeniedException {
-        resource.updateProperties(translator(), requestBody, resourceTriples);
+            final String requestBody,
+            final RdfStream resourceTriples)
+                    throws MalformedRdfException, AccessDeniedException {
+        if (resource instanceof NonRdfSourceDescription) {
+            // update the description instead
```

Comment:
It is surprising that you never saw FCREPO-1540, as it was included in the last three Tech Meetings:
https://wiki.duraspace.org/display/FF/2015-07-02+-+Fedora+Tech+Meeting
https://wiki.duraspace.org/display/FF/2015-07-09+-+Fedora+Tech+Meeting
https://wiki.duraspace.org/display/FF/2015-07-16+-+Fedora+Tech+Meeting

@escowles, are you ok with the approach in this PR?